ksql-rest-app/src/main/java/io/confluent/ksql/rest/server/computation/CommandRunner.java

```diff
@@ -91,6 +81,20 @@ public void close() {
     commandStore.close();
   }
 
+  public void fetchAndRunCommands() {
```

Comment:
This method is only visible due to testing. At the very least it should be made package-private, i.e, it def shouldn't be public.
An alternative would be to not make this class `Runnable` and instead use an `ExecutorService` in `KsqlRestApplication` to submit a `Runnable` with a fixed delay. You would then keep this public method on `CommandRunner` and remove the `run` method with the `while` loop.

Reply:
Made it package-private.